src/Frontend/Page/Comment.hs

```diff
@@ -50,7 +50,28 @@ instance ToHtml CommentWidget where
                         i_ [class_ "icon-flag"] nil
                         "melden"
         div_ [class_ "comment-replies"] $ do
-            for_ (comment ^. commentReplies) $ \reply -> do
-                div_ [class_ "comment-reply"] $ do
```

Comment:
good point.  but if that's the case, we can always revert to one widget type after styling.